.github/workflows/greetings.yml

```diff
@@ -0,0 +1,32 @@
+name: Greetings
+
+on: [pull_request_target]
```

Comment:
really need to run on `pull_requedt_target` not `pull_request` only?

Reply:
> TL;DR: Combining `pull_request_target` workflow trigger with an explicit checkout of an untrusted PR is a dangerous practice that may lead to repository compromise.
https://securitylab.github.com/research/github-actions-preventing-pwn-requests/